velox/dwio/dwrf/writer/Writer.h

```diff
@@ -31,6 +31,7 @@ namespace facebook::velox::dwrf {
 
 struct WriterOptions {
   std::shared_ptr<const Config> config = std::make_shared<Config>();
+  common::CompressionKind compressionKind = common::CompressionKind_MAX;
```

Comment:
Can you set this to the current default also please name it with a specific compression type.